openapi.yml

```diff
@@ -112,3 +115,268 @@ paths:
         in: path
         required: true
         description: Version of the package
+components:
+  schemas:
+    IndexData:
+      title: IndexData
+      type: object
+      properties:
+        version:
+          type: string
+        service.name:
+          type: string
+      required:
+        - version
+        - service.name
+    Category:
+      title: Category
+      type: object
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        count:
+          type: integer
+      required:
+        - id
+        - title
+        - count
+    Image:
+      title: Image
+      type: object
+      properties:
+        src:
+          type: string
+        title:
+          type: string
+        size:
+          type: string
+        type:
+          type: string
+      required:
+        - src
+    ProductRequirement:
+      title: Kibana
+      type: object
+      properties:
+        versions:
+          type: string
+    Requirement:
+      title: Requirement
+      type: object
+      properties:
+        Kibana:
+          $ref: '#/components/schemas/ProductRequirement'
+      required:
+        - Kibana
+    Dataset:
```

Comment:
I'll update this and other types now that https://github.com/elastic/package-registry/pull/514/files#diff-7dea786222588c32c19238bffffee9c2 is merged